From 48fd788215c5e63becfbd6aeff460b328da415b2 Mon Sep 17 00:00:00 2001
From: Vytautas Astrauskas <vastrauskas@gmail.com>
Date: Sun, 25 Jul 2021 18:50:52 +0200
Subject: [PATCH] Create a patch.

---
 build/gulpfile.extensions.js | 46 ++++--------------------------------
 1 file changed, 4 insertions(+), 42 deletions(-)

diff --git a/build/gulpfile.extensions.js b/build/gulpfile.extensions.js
index 60e41d2b020..e2bf3b030f0 100644
--- a/build/gulpfile.extensions.js
+++ b/build/gulpfile.extensions.js
@@ -24,48 +24,10 @@ const ext = require('./lib/extensions');
 const extensionsPath = path.join(path.dirname(__dirname), 'extensions');
 
 // To save 250ms for each gulp startup, we are caching the result here
-// const compilations = glob.sync('**/tsconfig.json', {
-// 	cwd: extensionsPath,
-// 	ignore: ['**/out/**', '**/node_modules/**']
-// });
-const compilations = [
-	'configuration-editing/build/tsconfig.json',
-	'configuration-editing/tsconfig.json',
-	'css-language-features/client/tsconfig.json',
-	'css-language-features/server/tsconfig.json',
-	'debug-auto-launch/tsconfig.json',
-	'debug-server-ready/tsconfig.json',
-	'emmet/tsconfig.json',
-	'extension-editing/tsconfig.json',
-	'git/tsconfig.json',
-	'github-authentication/tsconfig.json',
-	'github/tsconfig.json',
-	'grunt/tsconfig.json',
-	'gulp/tsconfig.json',
-	'html-language-features/client/tsconfig.json',
-	'html-language-features/server/tsconfig.json',
-	'image-preview/tsconfig.json',
-	'jake/tsconfig.json',
-	'json-language-features/client/tsconfig.json',
-	'json-language-features/server/tsconfig.json',
-	'markdown-language-features/preview-src/tsconfig.json',
-	'markdown-language-features/tsconfig.json',
-	'markdown-math/tsconfig.json',
-	'merge-conflict/tsconfig.json',
-	'microsoft-authentication/tsconfig.json',
-	'npm/tsconfig.json',
-	'php-language-features/tsconfig.json',
-	'search-result/tsconfig.json',
-	'simple-browser/tsconfig.json',
-	'testing-editor-contributions/tsconfig.json',
-	'typescript-language-features/test-workspace/tsconfig.json',
-	'typescript-language-features/tsconfig.json',
-	'vscode-api-tests/tsconfig.json',
-	'vscode-colorize-tests/tsconfig.json',
-	'vscode-custom-editor-tests/tsconfig.json',
-	'vscode-notebook-tests/tsconfig.json',
-	'vscode-test-resolver/tsconfig.json'
-];
+const compilations = glob.sync('**/tsconfig.json', {
+	cwd: extensionsPath,
+	ignore: ['**/out/**', '**/node_modules/**']
+});
 
 const getBaseUrl = out => `https://ticino.blob.core.windows.net/sourcemaps/${commit}/${out}`;
 
-- 
2.25.1

